src/target/riscv/program.h

```diff
@@ -28,10 +28,6 @@ struct riscv_program {
 	/* Number of 32-bit instructions in the program. */
 	size_t instruction_count;
 
-	/* Side effects of executing this program.  These must be accounted for
-	 * in order to maintain correct executing of the target system.  */
-	bool writes_xreg[RISCV_REGISTER_COUNT];
-
 	/* XLEN on the target. */
 	int target_xlen;
```

Comment:
Seems like this field is also unused. Please, consider removing it.

Reply:
You're right, thanks. Removed now.